### ISSUE TITLE
Clarification About GH-Pages

### ISSUE THREAD

**Issue by ccpandhare** [publiclab/image-sequencer#43]
I am not sure what we could put up on the `gh-pages` branch. As of now, ImageSequencer doesn't have a browser UI, although it does work on browser. So what should I update on the `gh-pages` branch? It won't be a very good demo, for sure...

**Comment by jywarren** [comment]
Do you think we need to make this part of the Interface phase, then? I
think ideally you could use parts of the UI that I had been developing,
though with the new way ImageSequencer works, it'd need some refactoring:
https://jywarren.github.io/image-sequencer/

For example, wed have to connect up the buttons again, but more
importantly, we'd have to create a new UI with HTML for each module as you
added modules, and also a handler to actually display the output of each
step.

**Comment by ccpandhare** [comment]
Maybe we could start early with the Interface phase and give it more priority as many things are actually on hold because of that. And shift other things to the near future. What do you say about this?

**Comment by jywarren** [comment]
I guess i'm excited about your progress to date, but we are still in the
first 3rd of the summer, so we needn't rearrange our big schedule if it's
disruptive. :-) Do you still feel that all the current set of tasks make
sense coming before the interface work? Or do you feel that there's enough
stability in the codebase now that you could productively pivot to the in
Interface phase, and that there are advantages to getting a web demo UI
running so people can follow your work (and begin to use it!?). I could go
either way, and am perfectly fine with holding off on interface work for
now if you think our original plan still makes sense. Thanks!

**Comment by ccpandhare** [comment]
I actually think the old plan does still make sense and that it's better to have majority of the functionality ready before we start developing the UI. That might save us time later on. We should maybe stick to this and develop the core modules with great thought so that they don't have to be reworked later on.

**Comment by jywarren** [comment]
OK! Thanks!
